en: add contents about modify tidb cluster config (#1033)
pingcap/docs-tidb-operator: en/upgrade-a-tidb-cluster.md

> If you modify configuration items by [dynamically modifying configuration online](https://docs.pingcap.com/tidb/stable/dynamic-config), after rolling upgrade, the configuration might be overwritten by CR.

1. Refer to [Configure TiDB components](configure-a-tidb-cluster.md#configure-tidb-components) to modify the component configuration in the `TidbCluster` CR of the cluster:

    {{< copyable "shell-regular" >}}

    ```shell
    kubectl edit tc ${cluster_name} -n ${namespace}
    ```

2. View the updating progress after the configuration is modified:

    {{< copyable "shell-regular" >}}

    ```shell
    watch kubectl -n ${namespace} get pod -o wide
    ```

>     After all the Pods are recreated and go to the `Running` state, the configuration is successfully modified.

Reviewer: ```suggestion
    After all the Pods are recreated and are in the `Running` state, the configuration is successfully modified.
```
